docs/unified-test-documentation/dasharo-security/20F-me-neuter.md

```diff
@@ -13,11 +13,144 @@
 1. Proceed with the
     [Generic test setup: OS boot from disk](../../generic-test-setup#os-boot-from-disk).
 
-## MNE001.001 ME neuter support (Ubuntu 22.04)
+## MNE001.001 ME disable option is available and has the correct default state
 
 **Test description**
 
-This test aims to verify that ME neuter function works correctly.
+This test aims to verify that the DME HAP disable state after flashing the
+platform with the Dasharo firmware is correct.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. While the DUT is booting, hold the `BIOS_SETUP_KEY` to enter the UEFI Setup
+    Menu.
+1. Enter the `Dasharo System Features` menu using the arrow keys and Enter.
+1. Enter the `Dasharo Security Options` submenu.
+1. Verify the `DME HAP disable` field.
```

Comment:
@sulewskiprzemyslaw @Pokisiekk `DME` ? We mentioned that there will be one option with 3 selectable entries.

The option name: `ME state`
You enter that menu and then you can select from list three options: `Enabled` `Soft-disabled` `HAP-disabled`

Reply:
@mkopec CC

Reply:
I've named the options as such - let me know if you want to change them @macpijan : 
![image](https://user-images.githubusercontent.com/3678707/198842403-f30bcfc7-0154-462a-bdc4-ec56ee80cbb3.png)

Reply:
Moreover the Intel Management Options submenu is located directly in the Dasharo System Features Menu, not inside the Dasharo Security Options menu.

Reply:
I have updated the documentation according to the given nomenclature.